widgetssdk/src/main/java/com/glia/widgets/call/CallController.kt

```diff
@@ -211,10 +226,12 @@ internal class CallController(
         queueIds: List<String>?,
         visitorContextAssetId: String?,
         mediaType: Engagement.MediaType?,
-        useOverlays: Boolean,
+        isEnableBubbleOutsideApp: Boolean,
+        isEnableBubbleInsideApp: Boolean,
         screenSharingMode: ScreenSharing.Mode
     ) {
-        sdkConfigurationManager.isUseOverlay = useOverlays
+        sdkConfigurationManager.isEnableBubbleOutsideApp = isEnableBubbleOutsideApp
+        sdkConfigurationManager.isEnableBubbleInsideApp = isEnableBubbleInsideApp
```

Comment:
In this case, I think that we will not be able to support the deprecated `useOverlay` on engagement start step. This is because overlay will only be configured on the SDK init step, not later. I need to think more, but this is what I think at the moment.

Reply:
What if we introduce breaking changes and get rid of this pain(`useOverlay`)?
Otherwise, we're adding a lot more complexity by adding these new attributes to this step.